app.go

```diff
@@ -463,12 +465,14 @@ func (app *app) loop() {
 			app.ui.draw(app.nav)
 		}
 	}
+
 }
 
 func (app *app) runCmdSync(cmd *exec.Cmd, pause_after bool) {
 	app.nav.previewChan <- ""
 	app.nav.dirPreviewChan <- nil
 
+	app.ui.screen.DisableFocus()
```

Comment:
I missed this yesterday, but this should probably be moved into the `app.ui.suspend` function, and similar for `EnableFocus` and `app.ui.resume`. I think in the general case it makes sense to change the focus reporting status when suspending/resuming the UI?

Reply:
I checked tcell and it already does disable/enable in its own suspend and resume code, so I remove those calls.